client-spark/spark-2/src/main/java/org/apache/spark/shuffle/celeborn/SortBasedShuffleWriter.java

```diff
@@ -211,7 +211,7 @@ private void fastWrite0(scala.collection.Iterator iterator) throws IOException {
 
   private void doPush() throws IOException {
     long start = System.nanoTime();
-    pusher.pushData();
+    pusher.pushData(false);
```

Comment:
why false here?

Reply:
I didn't intent to backport this functionality to spark-2 , shall I ?

Reply:
I'm OK with not backporting to spark-2, we can revisit this in future if it's required.